platform-includes/sourcemaps/overview/javascript.capacitor.mdx

```diff
@@ -0,0 +1,48 @@
+## Uploading Source Maps
+
+The easiest way to configure uploading source maps is by using the Sentry Wizard:
+
+<Include name="sourcemaps-wizard-instructions.mdx" />
+
+After choosing your Sentry project, the setup process will prompt you to select a framework. This step is crucial for configuring Sentry correctly. Make sure to choose the appropriate framework based on your project’s requirements.
```

Comment:
Since this guide is specifically for capacitor, can we provide more explicit instructions on what users should select?